src/status_im/chat/models/loading.cljs

```diff
@@ -20,17 +20,26 @@
 
 (fx/defn update-chats-in-app-db
   {:events [:chats-list/load-success]}
-  [{:keys [db]} new-chats]
+  [{:keys [db] :as cofx} new-chats]
   (let [old-chats (:chats db)
         chats (reduce (fn [acc {:keys [chat-id] :as chat}]
                         (assoc acc chat-id chat))
                       {}
                       new-chats)
-        chats (merge old-chats chats)]
-    {:db (assoc db :chats chats
-                :chats/loading? false)
-     :dispatch-n [[:chat/start-timeline-chat]
-                  [:start-profile-chat (get-in db [:multiaccount :public-key])]]}))
+        chats (merge old-chats chats)
+        chats-v (vals chats)
+        ens-names (get db :contacts/ens-names)]
+    (fx/merge cofx
+              {:db (assoc db :chats chats
+                          :chats/loading? false)
+               :dispatch-n [[:chat/start-timeline-chat]
+                            [:start-profile-chat (get-in db [:multiaccount :public-key])]]}
+              (doseq [chat chats-v]
```

Comment:
this is somehting really wrong, i don't understand this

Reply:
we cant fx/merge doseq

Reply:
why do we want to doseq all chats? i believe we need to verify only one entered name ?

Reply:
When closing / opening the app the ens-names on chats are lost so the main idea was to check ens-names and verify them again for each chat that was started through ENS search. I'm open to other approaches as this is not one I'm proud of.

Reply:
> When closing / opening the app the ens-names on chats are lost 

hm why are they lost ?

Reply:
If user is not saved to contacts after initializing a chat, ens name is lost. I'm trying to figure out why that's happening, but maybe we could investigate that in another issue and simplify this PR, because that is happening right now also when starting a chat with an ENS user from the search bar.